src/renderer/components/dialog-add-version.tsx

```diff
@@ -9,15 +9,16 @@ import {
 import { observer } from 'mobx-react';
 import * as path from 'path';
 import * as React from 'react';
+import * as fs from 'fs-extra';
```

Comment:
Since these are already in alphabetical order, let's keep them that way and move this up to be in alphabetical order.

Reply:
How about we introduce the [sort-import ](https://eslint.org/docs/latest/rules/sort-imports)rule in eslint rather than doing it manually everytime ?